docs/install_linux_mac.rst

```diff
@@ -286,8 +286,10 @@ Or, to install with PostgreSQL support:
 .. note::
 
   To install the development version, replace ``Red-DiscordBot`` in the above commands with the
-  following link:
-
+  link below. Please note that **dev is experimental and not intended for end users.** We will not
+  support anyone using the development version in any support channels. Downgrading after an
+  accidental install of dev is not possible without data loss.
```

Comment:
It might be better to say "not __always__ possible".

Reply:
Only replying cuz I’m nosey, I do agree but also disagree if we don’t want end users to use dev since it usually has breaking changes/incomplete changes etc and often leaves the user unable to downgrade, we shouldn’t imply that sometimes they will get support although it will happen we probably shouldn’t imply here, if we take an official stance that we will not help it should discourage people

Reply:
I'd prefer the more accurate wording here. It doesn't change that we don't provide support for development versions, only make it clear what risks might be associated with using that version.